lib/verdict/experiment.rb

```diff
@@ -63,6 +63,14 @@ def schedule_end_timestamp(timestamp)
     @schedule_end_timestamp = timestamp
   end
 
+  # Optional: Together with start and end timestamp, limits the experiment assignments within
```

Comment:
I think we can represent the experiment life cycle with a simple ascii that would contain "start date -> stop assignment date -> end date".